install_config/master_node_configuration.adoc

```diff
@@ -1203,8 +1203,21 @@ openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/openp
 
 The advanced audit feature provides several improvements over the
 xref:master-node-config-audit-config[basic audit functionality], including
-fine-grained events filtering and multiple output back ends. The following table
-contains additional options you can use.
+fine-grained events filtering and multiple output back ends. 
+
+To setup the advanced audit feature please add the following entries at the `openshift_master_audit_config` variable
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/oscp-audit/-oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/security/adv-audit.yaml","logFormat":"json"}
```

Comment:
Based on BZ1589063, how would one translate this into YAML-style inventory?

Reply:
good question. I haven't thought about this because in the inventory are the values in json.